html/signupsignin.html

```diff
@@ -8,16 +8,10 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
     <link rel="stylesheet" href="signupsignin.css">
```

Comment:
you did move files but you forgot to change the import path, so at the moment all styles should be broken.

Reply:
Moved to:     <link rel="stylesheet" href="../css/signupsignin.css">